_docs-sources/pipelines/security/machine-users.md

```diff
@@ -3,7 +3,13 @@ import TabItem from '@theme/TabItem';
 
 # Machine users
```

Comment:
This content is excellent, but it seems like the closest thing we have to "installation" instructions. Perhaps we should create a new sidebar section called "Initial setup" and list this there? Also, could you clarify when a user would need to do this? Or perhaps we should clarify explicitly that Gruntwork Pipelines is only available with DF, but will be available for standalone installation soon?